_query-dsl/joining/has-child.md

```diff
@@ -176,6 +176,140 @@ The response returns both brands:
 }
 ```
 
+## Retrieving inner hits
+
+To return child documents that matched the query, provide the `inner_hits` parameter:
```

Comment:
"matched" => "match"?

Reply:
I think it's better to leave as "matched" because the parent documents are returned as a consequence of the child documents matching the query.